cdk-deployment/app.py

```diff
@@ -8,6 +8,6 @@
 from validator_cdk.validator_cdk_stack import ValidatorCdkStack
 
 app = core.App()
-ValidatorCdkStack(app, "ValidatorCdkStack")
+ValidatorCdkStack(app, "STAC-Validator")
```

Comment:
A warning, I think this cause a lot of resources to be destroyed and recreated. For this it is trivial, but for projects with more resources, it can be problematic.